arch/arm/src/armv8-m/arm_tcbinfo.c

```diff
@@ -108,13 +108,15 @@ static const uint16_t g_reg_offs[] =
 
 const struct tcbinfo_s g_tcbinfo =
 {
-  .pid_off   = TCB_PID_OFF,
-  .state_off = TCB_STATE_OFF,
-  .pri_off   = TCB_PRI_OFF,
-  .name_off  = TCB_NAME_OFF,
-  .regs_off  = TCB_REGS_OFF,
-  .basic_num = 17,
-  .total_num = sizeof(g_reg_offs) / sizeof(g_reg_offs[0]),
+  .pid_off        = TCB_PID_OFF,
+  .state_off      = TCB_STATE_OFF,
+  .pri_off        = TCB_PRI_OFF,
+  .name_off       = TCB_NAME_OFF,
+  .stack_off      = TCB_STACK_OFF,
+  .stack_size_off = TCB_STACK_SIZE_OFF,
+  .regs_off       = TCB_REGS_OFF,
+  .basic_num      = 17,
+  .total_num      = sizeof(g_reg_offs) / sizeof(g_reg_offs[0]),
```

Comment:
note: can use `items(g_reg_offs)`

Reply:
@pkarashchenko here is fix: https://github.com/apache/nuttx/pull/9863/commits/71f0c952a5b20ccb41a9f4b95691e60e8a8a7950